_run/kube/kind.mk

```diff
@@ -12,6 +13,10 @@ PROVIDER_ENDPOINT ?= http://$(PROVIDER_HOST)
 
 INGRESS_CONFIG_PATH ?= https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/provider/kind/deploy.yaml
 
+ifeq (, $(shell which kind))
+$(error "No kind in $(PATH), consider install?")
```

Comment:
"run 'make devdeps-install' in the top-level directory to install"

Reply:
maybe put `devdeps-install` into some mk and include in any makefile that needs it?

Reply:
it only really needs to be run once.  i don't care all that much.